genapi.py

```diff
@@ -582,6 +582,17 @@ def __getattr__(self, key):
             return self._s.replace('\n', '\n' + ' '*n)
         return self.__dict__[key]
 
+def safe_output(cmd, shell=False, *args, **kwargs):
```

Comment:
btw,  you have an argument `shell` here, but don't use it (same with the other function)